apps/custom-ui-floating-window-demo/pages/api/auth/util.js

```diff
@@ -0,0 +1,30 @@
+export async function refreshAccessToken(refresh_token, user_id) {
```

Comment:
Please use camelCase in code if possible 
```suggestion
export async function refreshAccessToken(refreshToken, userId) {
```